bots/images/scripts/openshift.setup

```diff
@@ -42,17 +42,14 @@ printf "OPENSHIFT LISTENING ON LOCALHOST\n  $ ssh -NL 8443:localhost:8443 root@1
 ln -sf ../selinux/config /etc/sysconfig/selinux
 printf 'SELINUX=permissive\nSELINUXTYPE=targeted\n' > /etc/selinux/config
 setenforce 0
-systemctl stop firewalld
-dnf mark install iptables
-dnf -y remove firewalld
```

Comment:
any reason for dropping these two lines?

Reply:
I just stumbled over them when I was going over the `dnf` commands. Disabling firewalld is faster and more robust, and it then will still exist for possible experimentation.